### PR TITLE
Script to automate updating new node defs

### PR DESCRIPTION
**Pull request by jasonkarns**
This script runs the scraper, branches/commits/pushes and opens PRs for each new node def

### CODE REVIEW

**Comment by jawshooah** [comment]
Could be confusing to call it `update-definitions` right next to `nodenv update-version-defs`, which does something different. Perhaps `submit-new-definitions`?

**Comment by jasonkarns** [comment]
Agree in principle, though `update-version-defs` is a nodenv plugin. Whereas the npm scripts are:
`update-definitions`, and `write-definitions`.

Ignoring the plugin, what should the npm scripts be? `write-definitions` is basically just the scraper. We went with `write-definitions` instead of `scrape-definitions` for some reason that I don't recall.

Maybe `scrape-new-definitions` and `submit-new-definitions` for the pair of npm scripts? Does the fact that `submit-` also invokes the `scrape-*` make submit seem a bit too narrow?

**Comment by jawshooah** [comment]
I think it's an important enough distinction that it should be made explicit. The `scrape/submit` pair looks good to me

**Comment by jasonkarns** [comment]
updated to `scrape-definitions` and `submit-definitions` (the 'new' seemed unnecessary)

For a moment, I even thought about removing the `scrape-definitions` script from the package, since it's not really necessary. The submit script triggers the scraping anyway, and I don't intend for anyone to run the scrape script anymore. But I left it because I couldn't convince myself that no one would use it anymore. :shrug:

File: script/submit-definitions [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#
+# scrapes, commits, pushes, and pull-requests new node definitions
+#
+# * inspired by: https://github.com/jasonkarns/brew-publish *
+
+set -e
+
+while getopts ":r" opt; do
+  case $opt in
+  r) brew_remote="${OPTARG:-origin}" ;;
```
Comment (jawshooah) [inline script/submit-definitions]: Not particularly important, but this script doesn't have anything to do with `brew`. Should probably rename to `git_remote`

**Comment by jawshooah** [comment]
Aside from my nitpicking comment, LGTM :+1:

File: script/submit-definitions (hunk 1 above)
Comment (jasonkarns) [inline script/submit-definitions]: nice catch. copypasta fail :)

**Comment by jasonkarns** [comment]
i've now done enough tweaks that I'd like to see the script run and work again before I merge. So i'll leave this open until the next node release gives us an opportunity to run it.

**Comment by dudeche** [comment]
Looks like 4.3.1 is out.  Any chance that this is a good time to do it?

**Comment by jasonkarns** [comment]
@dudeche indeed! #119 was fully automated!

**Comment by dudeche** [comment]
Perfect!  Thanks!

**Pull request merged**